content/en/blog/_posts/bring-your-ideas-to-world-with-kubectl-plugins.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Bring your ideas to the world with kubectl plugins
+date: tbd
```

Comment:
Howdy @corneliusweig - this is the only hold out piece for the post and then we can LGTM it 😄

Reply:
Oh sorry, now this notification slipped through at my end :confused:

If I understand the process correctly, the release date will be determined by the blog team, right?

Reply:
Usually it is, we'll edit it as the last step before going live.

Reply:
I've added a date for tomorrow.